qcodes/instrument_drivers/Keysight/Keysight_E8267D.py

```diff
@@ -55,10 +60,20 @@ def __init__(self, name, address, step_attenuator=False, **kwargs):
                            get_cmd=':OUTP?',
                            set_cmd='OUTP {}',
                            get_parser=parse_on_off,
-                           # Only listed most common spellings idealy want a
-                           # .upper val for Enum or string
-                           vals=vals.Enum('on', 'On', 'ON',
-                                          'off', 'Off', 'OFF'))
+                           vals=on_off_validator)
+        self.add_parameter('IQmodulator',
```

Comment:
please also follow the naming recommendation [mentioned here](http://qcodes.github.io/Qcodes/examples/Creating%20Instrument%20Drivers.html?highlight=create_on_off#On/Off-parameters), so that this parameter should be called smth like `IQ_modulator_enabled`.

Reply:
Done